config.py

```diff
@@ -179,18 +179,19 @@ def create_config(config_file=None):
     logger.warning("No display configuration match.")
     sys.exit("Exiting...")
 
-# API URL for coingecko
-COINGECKO_URL_BASE = "https://api.coingecko.com/api/v3/"
+# API URL for binance websocket
+BINANCE_SOCKET = "wss://stream.binance.com:9443/ws/btcusdt@kline_1m" # every 2 seconds
```

Comment:
The "old" solution should be switchable in the config.
Many users want to use their own local currency